documentation/source/user_section/tutorials.rst

```diff
@@ -19,7 +19,11 @@ We provide the following hands-on tutorials for SCT's command-line tools.
    * :doc:`tutorials/registration-to-template/template-registration`
    * :doc:`tutorials/registration-to-template/shape-metric-computation`
    * :doc:`tutorials/registration-to-template/registering-additional-contrasts`
-   * :doc:`tutorials/registration-to-template/mtr-computation`
+
+#. :doc:`Non-template registration <tutorials/non-template-registration>`
+
+   * :doc:`tutorials/non-template-registration/mtr-computation`
+   * :doc:`tutorials/non-template-registration/contrast-agnostic-registration`
```

Comment:
Given that the "Registration" section is currently called "Registration to template", in order to distiguish non-template registration methods, I made a new section in the tutorials:

![image](https://user-images.githubusercontent.com/16181459/177579778-59dcafc6-1de4-4490-b69c-49c9eb6bc2b0.png)

The reason for this is because I believe the "MTR" tutorial shouldn't have been in the "Registration to template" section at all, as it doesn't use the PAM50 template.